draft-ietf-taps-interface.md

```diff
@@ -887,12 +887,27 @@ over these options.
 Name:
 : multipath
 
-This property specifies whether an application considers it useful to
-transfer data across multiple paths between the same end hosts. Generally,
-in most cases, this will improve performance (e.g., achieve greater throughput).
-One possible side-effect is increased jitter, which may be problematic for
-delay-sensitive applications. The recommended default is to Ignore this option.
+Type:
+: Enumeration
+
+This property specifies whether an application wants to take advantage of
+transferring data across multiple paths between the same end hosts. Using
+multiple paths allows connections to migrate between interfaces as
+availability and performance properties change. Possible values are:
+
+Disabled:
+: The connection will not attempt using multiple paths once established
```

Comment:
If we do the above, we can remove the disabled state

Reply:
How will I then turn it off? By setting prohibit for all other alternatives? That does not seem right. I think we would still need Disabled.